EngineLayer/Calibration/LabeledDataPoint.cs

```diff
@@ -17,7 +17,7 @@ public LabeledDataPoint(double experimentalMz, int scanNumber, double logTotalIo
             this.ScanNumber = scanNumber;
             this.LogTotalIonCurrent = logTotalIonCurrent;
             this.LogInjectionTime = logInjectionTime;
-            this.LogIntensity = logIntensity;
+            this.LogIntensity = logIntensity > 0 ? logIntensity : 0; //peaks with zero intensity can cause downstream crash
```

Comment:
What is the value of `logIntensity` when the intensity is zero? NaN?

Reply:
Negative Infinity